OrbitGl/App.cpp

```diff
@@ -1174,3 +1177,8 @@ void OrbitApp::InitializeClientTransactions() {
 void OrbitApp::FilterFunctions(const std::string& filter) {
   m_LiveFunctionsDataView->OnFilter(filter);
 }
+
+//-----------------------------------------------------------------------------
+void OrbitApp::CrashOrbitService(GetCrashRequest_CrashType crash_type) {
+  crash_manager_->CrashOrbitService(crash_type);
```

Comment:
Just call it on the thread_pool:
thread_pool_->Schedule([crash_type] { crash_manager_ ... });